README.md

```diff
@@ -18,6 +18,12 @@ Then add **hubot-youtube** to your `external-scripts.json`:
 ]
 ```
 
+Obtain a [Google Developer Console](https://console.developers.google.com) token. You will want the "Public" token rather than the OAuth credentials for this particular implementation. [Learn more](https://developers.google.com/console/help/new/?hl=en_US#generatingdevkeys)
+
```

Comment:
I would also recommend mentioning here to enable the 'YouTube Data API v3' under APIs.